invenio_pidstore/config.py

```diff
@@ -14,3 +14,13 @@
 This name will be used by the fetcher, to retrieve the record ID value from the
 JSON, and by the minter, to store it inside the JSON.
 """
+
+
+PIDSTORE_DATACITE_DOI_PREFIX = ''
+"""Provide a DOI prefix here."""
+
+PIDSTORE_DATACITE_DOI_OPTIONS = {
+    'suffix_length': 10,
+    'split_every': 5,
```

Comment:
This should be 8 and 4 instead of 10 and 5 to be compatible with https://blog.datacite.org/cool-dois/

Reply:
Discussed on video chat and we are good with 10 and 5.